### PR TITLE
Card adding process

### PR DESCRIPTION
**Pull request by filwas**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -4,29 +4,28 @@ import logo from "../public/FischLogo.svg";
...
 import { Card } from "./components/Card";
-import { useState } from "react";
+import { useRef, useState } from "react";
 
 function App() {
+  const [cardsAmount, setCardsAmount] = useState(1)
   const [cards, setCards] = useState([{}]);
 
   const handleAddCardButtonClick = () => {
+    setCardsAmount(cardsAmount+1)
```
Comment (Sumick) [inline src/App.tsx:14]: **nitpick:** I don't see any logic for adding card values. Why empty object?

File: src/App.tsx [hunk 2]
```diff
@@ -4,29 +4,28 @@ import logo from "../public/FischLogo.svg";
...
-  const handleDeleteCardButtonClick = (index:number) => {
-    //to be implemented
-  };
+  
 
   const renderCards = () => {
-    return cards.map((card, index) => (
-      <Card key={index} onDelete={() => handleDeleteCardButtonClick(index)} />
+    return cards.map(() => (
+      <Card cardsAmount={cardsAmount} setCardsAmount={setCardsAmount} />
```
Comment (Sumick) [inline src/App.tsx:21]: **nitpick:** I don't see how you display saved values in your cards. You are missing props here - `front`, and `back`

**Review by Sumick — CHANGES_REQUESTED** [review]
I don't see any logic for adding card values. Also `Card` is missing required props, so it works properly

File: src/App.tsx (hunk 2 above)
Comment (filwas) [reply src/App.tsx:21]: The values are stored in cards themselves. Each card has all of the code for editing it inside of it.

Comment (Sumick) [reply src/App.tsx:21]: But how are you gonna display values that come from backend? You should be able to pass values directly to Card component

Comment (filwas) [reply src/App.tsx:21]: Okay, gotcha.

**Pull request merged**